styles/styles.css

```
@@ -0,0 +1,395 @@
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("/fonts/ptsans-400.woff2") format("woff2");
+  font-display: swap;
+}
+
+@font-face {
+  font-family: "PT Sans";
+  font-style: normal;
+  font-weight: 700;
+  src: url("/fonts/ptsans-700.woff2") format("woff2");
+  font-display: swap;
+}
+
+body {
+    font-family: "PT Sans", sans-serif;
+    font-size: 22px;
+    line-height: 26px;
+    color: #000000;
+    background-color: #FFFFFF;;
+}
+.navigation-link{
+    font-size: 20px;
+    color:inherit;
+    font-weight: 700;
+    text-decoration: none;
+}
+.navigation-link-entrance{
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 20px;
+    align-items: center;
+    text-align: center;
+    text-transform: uppercase;
+    color: #FFFFFF;
+    background-color: #756157;
+    text-decoration: none;
+    border-radius: 4px;
+}
+.main-index{
+    color: #000000;
+    background-color: #FFFFFF;
+    background-image: url(/images/backgrounds/background-index.jpg);
```

Такая же ошибка, как и в подключении файла шрифта:
```suggestion
    background-image: url(../images/backgrounds/background-index.jpg);
```